dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresVectorStore.cs

```diff
@@ -110,7 +103,6 @@ public override Task DeleteCollectionAsync(string name, CancellationToken cancel
         return
             serviceKey is not null ? null :
             serviceType == typeof(VectorStoreMetadata) ? this._metadata :
-            serviceType == typeof(NpgsqlDataSource) ? this._dataSource :
```

Comment:
I believe it was a bug, we should not be exposing the data source by `GetService`